api/src/main.go

```diff
@@ -13,10 +13,15 @@ import (
 const (
 	PORT            = ":8080"
 	BUMP_INTERVAL   = 60 // 1 minute in seconds
+	TEST_INTERVAL   = 2
 	STORE_FILE_NAME = "store.json"
 )
 
-var gs GuildStore
+var (
+	TempTestInterval = 0    // used to set lower interval during testing
```

Comment:
Should this reflect the constant "TEST_INTERVAL"?

Reply:
the global variable `TEST_INTERVAL` has been removed.
This is now the only variable used when changing the interval for testing